### DISCUSSION TITLE
Dual Z-Axis Upgrade...

### DISCUSSION THREAD

**Discussion by Dooti-2** [mriscoc/Ender3V2S1#58] [Q&A]
So I bought the kit...installed it...and It's not awesome.  I have seen some firmware compiled to enable settings for this but the version I have has the Z-cable jumped and going into the board (4.2.7) with one plug...vs. two like maybe some others have?  Either way, it kinda sucks and I may just take it off if you all haven't seen it work well with this AWESOME firmware.

**Comment by Dooti-2** [comment]
uhh, so I had the lead screw a bit too tight + misaligned on the carriage / bracket.  I adjusted it and aside from it being slightly louder when doing ABL / building a mesh it's pretty slick.

**Comment by JoeMaxey** [comment]
Helpful, thanks for sharing. Waiting to install my dual z kit til this weekend.

**Comment by hyi1233** [comment]
Double Motor Z axis, not very good, do a good plan is two screw, synchronous belt synchronization, there will be no double motor pulse signal caused by motor synchronization problem ~
